source/getting_started/install_macs.md

```diff
@@ -0,0 +1,86 @@
+# Prerequisites for Macs
+
+You must have the following set up on your mac laptop:
+
+- administrator rights on your laptop
+- text editor
+- [Xcode](https://developer.apple.com/xcode/) command line interface tools [external link]
+- [Ruby](https://www.ruby-lang.org/en/) [external link]
+- [Bundler](https://bundler.io/) [external link]
+- [Middleman](https://middlemanapp.com/) static site generator [external link]
+
+You must also have a [GitHub account](https://github.com/) [external link] with access to alphagov at `https://github.com/alphagov/paas-tech-docs`.
+
+## Administrator rights on your laptop
+
+Email [sd-community@digital.cabinet-office.gov.uk](mailto:sd-community@digital.cabinet-office.gov.uk) to request administrator rights on your laptop.
+
+## Install text editor
+
+You should install a text editor to edit your content, for example:
+
+- [Atom](https://atom.io/) [external link]
+- [SublimeText](https://www.sublimetext.com/) [external link]
+- [Notepad++](https://notepad-plus-plus.org/) [external link]
+- [TextWrangler](https://www.barebones.com/products/textwrangler/) [external link]
+
+You can edit your content using an internal text editor such as [VIM](https://www.vim.org/) [external link].
+
+## Install Xcode command line interface tools
+
+1. Go to the Managed Software Centre on your Mac.
```

Comment:
I'm not sure other departments will use MSC; GDS is using jamf now for example

Reply:
have amended to state the assumption of MSC